scripts/release

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
when/how does this script get run

Reply:
locally similar to what we do for nebula releases